src/panels/lovelace/cards/energy/hui-energy-solar-graph-card.ts

```diff
@@ -110,84 +120,98 @@ export class HuiEnergySolarGraphCard
   }
 
   private _createOptions = memoizeOne(
-    (start: Date, end: Date, locale: FrontendLocaleData): ChartOptions => ({
-      parsing: false,
-      animation: false,
-      scales: {
-        x: {
-          type: "time",
-          suggestedMin: start.getTime(),
-          suggestedMax: end.getTime(),
-          adapters: {
-            date: {
-              locale: locale,
+    (start: Date, end: Date, locale: FrontendLocaleData): ChartOptions => {
+      const dayDifference = differenceInDays(end, start);
+      return {
+        parsing: false,
+        animation: false,
+        scales: {
+          x: {
+            type: "time",
+            suggestedMin: (dayDifference > 2
+              ? addHours(start, -11)
+              : start
+            ).getTime(),
+            suggestedMax: (dayDifference > 2
+              ? addHours(end, -11)
```

Comment:
Should this be plus 12?

Reply:
Nope, 12 gave an extra grid line, 11 looked perfect

Reply:
Right, I meant  that this is - and not +.

Reply:
Aaaahh... No, it does not 😄 now the day bars are nice in the middle
![image](https://user-images.githubusercontent.com/5662298/128777186-4f087486-8f89-4a54-8d5c-80bf9c0f93e1.png)